GradienceTeam/Gradience
# Move modules to new backend/frontend directories

# Description

This is a yet another refactoring PR (I promise this will be the last _major_ one), this time I based it on a codebase structure used by Bottles project. The new structure should be more readable, and easier to navigate. Also, as a nice small bonus, I've included `struct.md` files inside `frontend/` and `backend/` directories with small descriptions for subdirectories.

## Type of change



## Changelog 

- Rename `utils/css.py` to `utils/css_parser.py` and `utils/custom_presets.py` to `utils/preset_downloader.py`,
- Move modules to new backend/frontend directories,
- Change module name from `utils.py` to `common.py`,
- Format some modules for better readability

## Testing


### How to test the changes

Just check if everything is working as expected in every build method (I've tested this PR with manual and Flatpak methods)
